Simplified/KeychainStoredVariable.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  KeychainStoredVariable.swift
+//  SimplyE
+//
+//  Created by Jacek Szyja on 22/05/2020.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+class KeychainVariable<VariableType> {
+  let transaction: KeychainVariableTransaction
+  var key: String {
+    didSet {
+      guard key != oldValue else { return }
+      alreadyInited = false
+    }
+  }
+
+  var alreadyInited = false
```

Comment:
alreadyInited is not correlated with thread safety. `KeychainVariable` class has a variable `cachedValue`, which should store in-memory the latest value written to, or read from keychain (so that we don't access secure enclave, which is quite expensive, every time we want to draw a cell for example). Value that currently is stored in a keychain may be a nil value, and we don't want to ask for it over and over again. `alreadyInited` was created to solve this exact case, that we already asked for a keychain entry, and it was nil. Other case is when the key used to access keychain changed, the cached value would contain old data in such case and alreadyInited would inform about it.

Reply:
gotcha, sounds good. This was not clear to me. If you could add a documentation comment for this that would be great!